indy_common/types.py

```diff
@@ -73,6 +76,17 @@ class ClientGetNymOperation(MessageValidator):
     )
 
 
+class ClientNYMOperation(PClientNYMOperation):
+    schema = (
+        (TXN_TYPE, ConstantField(NYM)),
+        (ALIAS, LimitedLengthStringField(max_length=ALIAS_FIELD_LIMIT, optional=True)),
+        (VERKEY, VerkeyField(optional=True, nullable=True)),
+        (TARGET_NYM, DestNymField()),
+        (ROLE, RoleField(optional=True)),
+        (DIDDOCCONTENT, LimitedLengthStringField(max_length=RAW_FIELD_LIMIT, optional=True)),
```

Comment:
Nitpick: I think we should make `DIDDOCCONTENT` be `DIDDOC_CONTENT`; I think this will match the convention followed by the other constants a bit better.